examples/Robot1/main.cpp

```diff
@@ -9,6 +9,11 @@
 #include <random>
 #include <chrono>
 
+#ifdef _WIN32
```

Comment:
Wouldn't`#include <cmath>` help here? if not, at least chaning to `#ifndef M_PI` would be preferred here..